virtme/commands/run.py

```diff
@@ -1301,19 +1306,40 @@ def do_it() -> int:
             mount_tag="virtme.guesttools",
         )
         export_virtfs(qemu, arch, qemuargs, virtfs_config)
+        initsh = [
+            "mount -t tmpfs run /run",
+            "mkdir -p /run/virtme/guesttools",
+            "/bin/mount -n -t 9p -o ro,version=9p2000.L,trans=virtio,access=any "
+            + "virtme.guesttools /run/virtme/guesttools",
+        ]
+        if args.systemd:
+            kernelargs.append("console=ttyS0")
+            # TODO: implies --rw
+            initsh.append(
+                "cp /usr/lib/systemd/system/serial-getty@.service /etc/systemd/system"
```

Comment:
To avoid needing `--rw`, we could keep a patched .service file in `virtme/guest/virtme-getty-serial.service` and use something like:

```
kernelargs.append("systemd.unit=virtme-getty-serial.service")
initsh.append("SYSTEMD_UNIT_PATH=/run/virtme/guesttools: exec /sbin/init")
```